app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    @user = User.find_by_email(params[:email])
+    if @user.authenticate(params[:password])
+      session[:user_id] = @user.id
+      flash[:success] = "Successfully logged in!"
+      redirect_to dashboard_path
+    end
```

Comment:
That's a a good call. I'll make sure to write tests and add the error messages before we merge it with master